libs/features/mas/src/variants/simplified-pricing-express.js

```diff
@@ -65,58 +70,162 @@ export class SimplifiedPricingExpress extends VariantLayout {
+                ),
+            ) || 0;
+
+        if (height > maxMinHeight) {
+            container.style.setProperty(
+                elMinHeightPropertyName,
+                `${height}px`,
+            );
+        }
+    }
+
+    syncHeights() {
+        if (this.card.getBoundingClientRect().width === 0) {
+            return;
+        }
+
+        const descriptionSlot = this.card.querySelector('[slot="body-xs"]');
+        if (descriptionSlot) {
+            descriptionSlot.offsetHeight;
+
+            const height = descriptionSlot.offsetHeight;
+            this.updateCardElementMinHeightValue(height, 'description');
+        }
+
+        const priceSlot = this.card.querySelector('[slot="price"]');
+        if (priceSlot) {
+            priceSlot.offsetHeight;
+
+            const height = priceSlot.offsetHeight;
+            this.updateCardElementMinHeightValue(height, 'price');
+        }
+    }
+
+    clearHeightVariables() {
+        const container = this.getContainer();
+        if (!container) return;
+
+        const prefix = `--consonant-merch-card-${this.card.variant}`;
+        ['description', 'price'].forEach(name => {
+            container.style.removeProperty(`${prefix}-${name}-height`);
+        });
+    }
+
+    async waitForPrices() {
+        if (this.card.prices?.length) {
+            await Promise.all(this.card.prices.map(price => price.onceSettled?.()));
+        }
+    }
+
+    async syncAllCardsInContainer(container) {
+        const cards = container.querySelectorAll('merch-card[variant="simplified-pricing-express"]');
+
+        if (!cards.length) return;
+
+        const prefix = `--consonant-merch-card-${this.card.variant}`;
+        ['description', 'price'].forEach(name => {
+            container.style.removeProperty(`${prefix}-${name}-height`);
+        });
+
+        await Promise.all(
+            Array.from(cards).flatMap(card =>
+                card.prices?.map(price => price.onceSettled?.()) || []
+            )
+        );
+
+        requestAnimationFrame(() => {
+            cards.forEach(card => card.variantLayout?.syncHeights?.());
+        });
+    }
+
+    async forceRemeasure() {
+        const container = this.getContainer();
+        if (container) {
+            await this.syncAllCardsInContainer(container);
+        }
+    }
+
+    async postCardUpdateHook() {
+        if (!this.card.isConnected) return;
+
+        await this.card.updateComplete;
+        await this.waitForPrices();
+
+        if (!!isDesktop()) {
+            const container = this.getContainer();
+            if (!container) return;
+
+            const prefix = `--consonant-merch-card-${this.card.variant}`;
+            const hasExistingVars = container.style.getPropertyValue(`${prefix}-description-height`) ||
+                                   container.style.getPropertyValue(`${prefix}-price-height`);
+
+            if (!hasExistingVars) {
+                await this.syncAllCardsInContainer(container);
```

Comment:
I still don't get the container logic. Each card will update those values and they will end up being in sync.

Reply:
This logic determines whether to sync all cards or just the current card based on whether CSS variables already exist in the container. WIthout it there will be more DOM operations.